docs/buildah-build.1.md

```diff
@@ -34,6 +34,8 @@ Add a custom host-to-IP mapping (host:ip)
 
 Add a line to /etc/hosts. The format is hostname:ip. The **--add-host** option can be set multiple times. Conflicts with the --no-hosts option.
 
+Note: Following flag supports a special `host-gateway` value that resolves to the internal IP address of the host. This is useful when you want containers to connect to services running on the host machine. Usage format (host:host-gateway).
```

Comment:
Is "host" in "(host:host-gateway)" also literal, like "host-gateway"?  Is this syntax also accepted by `buildah run`?

Reply:
No `host` is not a literal string, it can be any hostname. I have added `<>` to convey that it is a parameter.

> Is this syntax also accepted by buildah run?

I don't think `--add-host` exists for buildah run.